_client/src/Network/NetworkService.cpp

```diff
@@ -16,8 +16,7 @@ void NetworkService::sendRequest(QString endpoint, Method method, const Dataset
         SPDLOG_ERROR("NetworkService::sendRequest serializer was not set");
         return;
     }
-
-    QUrl fullUrl(m_apiUrl + endpoint);
+    QUrl fullUrl{m_apiUrl + endpoint + "/" + resource_id};
```

Comment:
@VolodymyrBilenko needs refactoring.

We could just overload this method to have different params or separate responsibility of creating url to different overloaded methods.